cli/bpmetadata/cmd.go

```diff
@@ -368,3 +420,20 @@ func UnmarshalMetadata(bpPath string) (*BlueprintMetadata, error) {
 
 	return &bpObj, nil
 }
+
+func buildUIInputFromVariables(vars []BlueprintVariable, input *BlueprintUIInput) {
```

Comment:
I started out with that but then moved it back because it was just one lonely function in there. But I think it makes sense if you're thinking along the same lines as well. Moved.